src/main/scala/viper/gobra/translator/util/Core.scala

```diff
@@ -62,11 +63,19 @@ object Core {
     } yield x.copy(args = args)(x.info)
   }
 
+  def deepCore(x: in.ClosureSpec): Core[in.ClosureSpec] = {
+    for {
+      paramValues <- vector(x.paramValues.map(y => option(y.map((core)))))
```

Comment:
nitpick: duplicated pair of parentheses around `core`

Reply:
Parentheses removed